tests/lookup/tests.py

```diff
@@ -97,7 +97,11 @@ def test_iterator(self):
 
     def test_count(self):
         # count() returns the number of objects matching search criteria.
-        self.assertEqual(Article.objects.count(), 7)
+        with self.assertNumQueries(1) as ctx:
+            self.assertEqual(Article.objects.count(), 7)
+        sql = ctx.captured_queries[0]['sql']
+        self.assertTrue(sql.startswith('SELECT COUNT(*) '), sql)
```

Comment:
I couldn't decide whether this should be in the aggregates or the lookups. Also I couldn't find a precedent for checking the SQL the ORM outputs, maybe due to database difficulties - this fails on sqlite because the returned query is not the formatted sql, but `'QUERY = u'SELECT COUNT(*) AS "__count" FROM "lookup_article"' - PARAMS = ()'`.